lightkurve/targetpixelfile.py

```diff
@@ -404,18 +405,33 @@ def create_threshold_mask(self, threshold=3):
         in a robust way by multiplying the Median Absolute Deviation (MAD)
         with 1.4826.
 
+        If the thresholding method yields multiple contiguous regions, then
+        only the region closest to the (col, row) coordinate specified by
+        `reference_pixel` is returned, e.g. `reference_pixel=(0, 0)` will pick
+        the region closest to the bottom left corner.
+        By default, the region closest to the center of the mask will be
+        returned. If `reference_pixel='all'` then all regions will be returned.
```

Comment:
Perhaps all contiguous regions should be returned when `reference_pixel=None`.

'Multiple' reference pixels doesn't seem to make a tone of sense to me. No reference pixel it seems intuitive that it would pass back all regions.

This is a nitpick.